src/Calculator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Acme;
+
+class Calculator
+{
+    public $displayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyaaaayyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy;
+
+    function div(int $a, int $b): int {
```

Comment:
<sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
error - Opening brace should be on a new line